qcodes/instrument_drivers/Keysight/keysight_e4980a.py

```diff
@@ -236,6 +237,32 @@ def __init__(self,
                       "the auto range function OFF."
         )
 
+        self.add_parameter(
+            "dc_bias_enabled",
+            get_cmd=self._get_dc_bias_state,
+            set_cmd=self._enable_dc_bias,
+            vals=Bool(),
+            val_mapping=create_on_off_val_mapping(on_val="ON", off_val="OFF"),
+            docstring="Enables DC bias. DC bias is automatically turned off "
+                      "after recalling the state from memory."
+        )
+
+        self.add_parameter(
+            "dc_bias_voltage_level",
+            get_cmd=self._get_dc_bias_voltage_level,
+            set_cmd=self._set_dc_bias_voltage_level,
+            unit="V",
+            docstring="Sets the DC bias voltage. Setting does not implicitly "
+                      "turn the DC bias ON."
+        )
+
+        self.add_parameter(
+            "measurement_time",
+            get_cmd=":APERture?",
+            set_cmd=self._set_measurement_time,
+            docstring="Sets the measurement time mode and the averaging rate."
```

Comment:
typo?
```suggestion
            docstring="Sets the measurement time and the averaging rate."
```

Reply:
No, it is not typo. It is the mode which is being set.